### PR TITLE
Release 2.22.0

### PR DESCRIPTION
**Pull request by marthamareal**

### CODE REVIEW

File: _posts/2022-11-18-geoserver-2-22-0-released.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+[GEOS-10714](https://osgeo-org.atlassian.net/browse/GEOS-10714) DefaultGeoServerFacade throws ConcurrentModificationException for workspace settings and services
+
+[GEOS-10729](https://osgeo-org.atlassian.net/browse/GEOS-10729) Concurrent access on data access rules \(authorization\) can lead to loss of configured catalog mode, and lost rules
+
+[GEOS-10731](https://osgeo-org.atlassian.net/browse/GEOS-10731) GWC variable Parameterization does not work with geoserver-environment.properties due to the bean initialization order
+
+[GEOS-10736](https://osgeo-org.atlassian.net/browse/GEOS-10736) OSEO product creation via REST API fails if the product id starts with a valid ISO date
+
+[GEOS-10737](https://osgeo-org.atlassian.net/browse/GEOS-10737) GeoCSS misses support for labelInFeatureInfo and labelAttributeName vendor options
+[GEOS-10741](https://osgeo-org.atlassian.net/browse/GEOS-10741) Remove deprecated YUI usage
```
Comment (mprins) [inline _posts/2022-11-18-geoserver-2-22-0-released.md:71]: add an empty line above
```suggestion

[GEOS-10741](https://osgeo-org.atlassian.net/browse/GEOS-10741) Remove deprecated YUI usage
```

**Review by mprins — CHANGES_REQUESTED** [review]
need an empty line

File: _posts/2022-11-18-geoserver-2-22-0-released.md (hunk 1 above)
Comment (marthamareal) [reply _posts/2022-11-18-geoserver-2-22-0-released.md:72]: updated

**Review by mprins — APPROVED** [review]

**Comment by jodygarnett** [comment]
I should have some headings to add for geotools features; there was a new YSLD support for sld title / description (for example).

File: _posts/2022-11-18-geoserver-2-22-0-released.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+GeoServer [2.22.0](/release/2.22.0/) release is now available with downloads ([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/geoserver-2.22.0-bin.zip/download), [war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/geoserver-2.22.0-war.zip/download), [windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/GeoServer-2.22.0-winsetup.exe/download)), along with [docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/geoserver-2.22.0-htmldoc.zip/download) and [extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/extensions/).
+
+This is a stable release of the GeoServer 2.22.x series, made in conjunction with GeoTools 28.0 
+and GeoWebCache 1.22.0.
+
+All major new features have been described in the [Release Candidate (RC) Blog post](https://geoserver.org/announcements/2022/10/19/geoserver-2-22-RC-released.html).
+
+Thanks to Martha Nagginda (GeoSolutions) and Andrea Aime (GeoSolutions) for making this release.
+
+### Improvements and Fixes
```
Comment (jodygarnett) [inline _posts/2022-11-18-geoserver-2-22-0-released.md:25]: Please use 2.22-RC release announcement as starting point (it includes visual reference of new features and functionality).

**Review by jodygarnett — CHANGES_REQUESTED** [review]
Migrate content from 2.22-RC announcement. We also try and thank folks who tested the release candidate.

File: _posts/2022-11-18-geoserver-2-22-0-released.md (hunk 2 above)
Comment (marthamareal) [reply _posts/2022-11-18-geoserver-2-22-0-released.md]: @jodygarnett For the notes in the 2.22-RC, i linked it on line 21 mentioning a read of the major features, than copying them again. what do you think?

Comment (marthamareal) [reply _posts/2022-11-18-geoserver-2-22-0-released.md]: I have added the notes from 2.22-RC as suggested.

Comment (jodygarnett) [reply _posts/2022-11-18-geoserver-2-22-0-released.md]: Perfect thank you - folks need the visual.

**Pull request merged**